httpclient/api_client.go

```diff
@@ -257,8 +258,8 @@ func (c *ApiClient) recordRequestLog(
 		RequestBody:              requestBody,
 		ResponseBody:             responseBody,
 		DebugHeaders:             c.config.DebugHeaders,
+		DebugAuthorizationHeader: c.config.DebugAuthorizationHeader,
```

Comment:
you're missing the `X-Databricks-Azure-SP-Management-Token` and `X-Databricks-GCP-SA-Access-Token` headers in https://github.com/databricks/databricks-sdk-go/blob/main/logger/httplog/round_trip_stringer.go#L33-L35

Reply:
I think you reviewed a stale version but I added this in the last commit